.gitignore

```diff
@@ -1 +1,6 @@
-/bazel-*
+*.pyc
+.cache
+sysl.egg-info
+dist
+build
+.eggs
```

Comment:
Minor quibble: all non-empty files should end in a newline. Ditto for other files below.

Reply:
(It's not evident in PR view, but you can see missing-newline indicators in files view.)

Reply:
Totally agreed. I've already added a `.editorconfig` file. I wanted to wait for the next PR to run pylint (PEP 8 standard) and editorconfig over all source files.